azure/defaults.go

```diff
@@ -65,6 +65,11 @@ const (
 	ControlPlaneNodeGroup = "control-plane"
 )
 
+const (
+	// BootstrapExtensionCommand is the command that runs on the Boostrap VM extension to check for bootstrap success
+	BootstrapExtensionCommand = "for i in $(seq 1 240); do test -f /run/cluster-api/bootstrap-success.complete && break; if [ $i -eq 240 ]; then return 1; else sleep 5; fi; done"
```

Comment:
can we assign 240 to a constant for more clarity?
maybe more notes on `/run/cluster-api/bootstrap-success.complete` - where this comes from and what it does?